packages/e2e/cypress/integration/common/storagebrowser.ts

```diff
@@ -3,7 +3,7 @@ import { When } from '@badeball/cypress-cucumber-preprocessor';
 When(
   'I drag and drop a file into the storage browser with file name {string}',
   (fileName: string) => {
-    cy.get('[data-testid="storage-browser-table"').trigger('drop', {
+    cy.get('[class="storage-browser__drop-zone"').trigger('drop', {
```

Comment:
nit: is this change needed if u are not running the test atm?

Reply:
It'll eventually need to be fixed and the target will be this still I think